packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -1,5 +1,133 @@
+import copy
 import json
+import uuid
 
+from dcos_test_utils import marathon
 
+TEST_APP_NAME_FMT = 'integration-test-{}'
+
+# make the expanded config available at import time to allow determining
+# which tests should run before the test suite kicks off
 with open('/opt/mesosphere/etc/expanded.config.json', 'r') as f:
     expanded_config = json.load(f)
+
+
+def marathon_test_app(
+        host_port: int=0,
+        container_port: int=None,
+        container_type: marathon.Container=marathon.Container.NONE,
+        network: marathon.Network=marathon.Network.HOST,
+        healthcheck_protocol: marathon.Healthcheck=marathon.Healthcheck.HTTP,
+        vip: str=None,
+        host_constraint: str=None):
+    """ Creates an app definition for the python test server which will be
+    consistent (i.e. deployable with green health checks and desired network
+    routability). To learn more about the test server, see in this repo:
+    ../packages/dcos-integration-test/extra/util/python_test_server.py
+
+    Args:
+        host_port: port that marathon will use to route traffic into the
+            test server container. If set to zero, then marathon will assign
+            a port (which is referenced by index).
+        container_port: if network is BRIDGE, then the container can have a
+            port remapped inside the container. In HOST or USER network, the
+            container port must be the same as the host port
+        container_type: can be NONE (default Mesos runtime), MESOS (the UCR),
+            or DOCKER
+        health_check_protocol: can be MESOS_HTTP or HTTP
+        vip: either named or unnamed VIP to be applied to the host port
+        host_constraint: string representing a hostname for an agent that this
+            app should run on
+
+    Return:
+        (dict, str): 2-Tuple of app definition (dict) and app ID (string)
+    """
+    if network == marathon.Network.BRIDGE:
+        assert container_type == marathon.Container.DOCKER, \
+            'BRIDGE network mode only supported for DOCKER container type'
+        if container_port is None:
+            # provide a dummy value for the bridged container port if user is indifferent
+            container_port = 8080
+    else:
+        assert container_port is None or container_port == host_port, 'Cannot declare a different host and '\
+            'container port outside of BRIDGE network'
+        container_port = host_port
+    if network == marathon.Network.USER:
+        assert host_port != 0, 'Cannot auto-assign a port on USER network!'
+
+    test_uuid = uuid.uuid4().hex
+    app = copy.deepcopy({
+        'id': TEST_APP_NAME_FMT.format(test_uuid),
+        'cpus': 0.1,
+        'mem': 32,
+        'instances': 1,
+        'cmd': '/opt/mesosphere/bin/dcos-shell python '
```

Comment:
I'd be surprised if this works with the docker containerizer. I was under the impression that the host filesystem is not visible when running with it.